ui/src/api/utils.js

```diff
@@ -80,3 +82,27 @@ export const siblingRelationPayload = (relationType, extra, second) => {
     ...extra,
   };
 };
+
+export const formatPrice = price => {
+  if (!price) return null;
+
+  const value = price.value.toFixed(2);
+
+  switch (price.currency) {
```

Comment:
Maybe evaluate https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat instead?

Reply:
It seems to be a better solution than rolling our own. All the major browsers have had support for it since at least 2016 so for me it looks good.